src/v/cloud_storage/partition_manifest.cc

```diff
@@ -403,6 +403,11 @@ size_t partition_manifest::segments_metadata_bytes() const {
     return _segments.inflated_actual_size().second;
 }
 
+size_t partition_manifest::estimate_serialized_size() const {
+    constexpr auto bytes_per_segment = 10;
+    return _segments.size() * bytes_per_segment;
+}
```

Comment:
How accurate can we expect this to be? Could you add a test for it?

Reply:
same here, it's an estimate, not precise value
if the estimate is wrong we will actually use less bandwidth than advertised (no problem) or a bit more (the scheduler will suspend archiver for a bit longer in this case)
the code path is used in tests
it'd be also totally fine to not to try to account for this at all

Reply:
>the code path is used in tests

I don't see any tests in this PR? Or do you mean a later PR